### DISCUSSION TITLE
Snippets

### DISCUSSION THREAD

**Discussion by Bruno0798** [microsoft/vscode-cpptools#11525] [Q&A]
Hey everyone, I have installed the Microsoft extension for IntelliSense, and it offers some cool snippets, but I would like to change the main snippet, because I don't want to put the const before argv, is there any way to edit.

Thanks in advance.

**Comment by TryerGit** [comment]
Where snippets exactly reside in VSCode is a simple question with a somewhat complicated answer. See here: https://stackoverflow.com/questions/69347454/where-do-code-snippets-reside-in-visual-studio-code

Best bet would be to create a user snippet which does what you want. Then, you can tab-choose that and you can get your required code expanded.
